Fixed some typos in README and removed e.g. from example usage (#2)
bshall/VectorQuantizedCPC: README.md

    Note: `in_dir` must be the path to the `2019` folder. 
    For `dataset` choose between `2019/english` or `2019/surprise`.
    Other datasets will be added in the future.
    Example usage:
    ```
    python preprocess.py in_dir=../datasets/2020/2019 dataset=2019/english
    ```
    
## Training
   

> 1.  Train the VQ-CPC model (pretrained weights are given above):

Reviewer: If you don't mind, could we change this to:
```suggestion
1.  Train the VQ-CPC model (or download pretrained weights [here](https://github.com/bshall/VectorQuantizedCPC/releases/tag/v0.1):
```

containing a list of items with a) the path (relative to `in_dir`) of the source `wav` files;
b) the target speaker (see `datasets/2019/english/speakers.json` for a list of options);
and c) the target file name.
Example usage:
```
python convert.py cpc_checkpoint=checkpoints/cpc/english2019/model.ckpt-25000.pt vocoder_checkpoint=checkpoints/vocoder/english2019/model.ckpt-150000.pt in_dir=../datasets/2020/2019 out_dir=submission/2019/english/test synthesis_list=datasets/2019/english/synthesis.json in_dir=../../Datasets/2020/2019 dataset=2019/english
```

> Voice conversion samples will be available soon.

Reviewer: Might as well add another link to the samples too? It's nice to have the links near to the different sections as well.
```suggestion
Voice conversion samples are available [here](https://bshall.github.io/VectorQuantizedCPC/)
```

Author: I've made the changes below (I didn't realise you could commit it directly!).

```
python convert.py cpc_checkpoint=checkpoints/cpc/english2019/model.ckpt-25000.pt vocoder_checkpoint=checkpoints/vocoder/english2019/model.ckpt-150000.pt in_dir=../datasets/2020/2019 out_dir=submission/2019/english/test synthesis_list=datasets/2019/english/synthesis.json in_dir=../../Datasets/2020/2019 dataset=2019/english
```
Voice conversion samples are available

> [here](https://bshall.github.io/VectorQuantizedCPC/).

Author: I'm checking out how to leave comments :)
